src/manager.ts

```diff
@@ -1,16 +1,31 @@
 import { addons } from "@storybook/manager-api";
 import { Addon_TypesEnum } from "@storybook/types";
 import { Tool, Panel, PanelTitle } from "@/components/Addons";
-import { ADDON_ID, ADDON_ID_FOR_PARAMETERS, PANEL_ID, TOOL_ID } from "./consts";
+import {
+  ADDON_ID,
+  ADDON_ID_FOR_PARAMETERS,
+  PANEL_ID,
+  PLAYGROUND_STORY_PREPARED,
+  TOOL_ID,
+} from "./consts";
 import { PlaygroundParameters } from "@/types";
+import { STORY_PREPARED } from "@storybook/core-events";
+import { getStoryId } from "@/utils";
 
-addons.register(ADDON_ID, ({ getCurrentParameter }) => {
+addons.register(ADDON_ID, ({ getCurrentParameter, on, emit, resolveStory }) => {
   function getPlaygroundStoryId() {
     const { storyId } =
       getCurrentParameter<PlaygroundParameters>(ADDON_ID_FOR_PARAMETERS) || {};
     return storyId;
   }
 
+  on(STORY_PREPARED, ({ id: preparedStoryId }) => {
+    const resolvedStory = resolveStory(getPlaygroundStoryId());
```

Comment:
Suggestion: add `isPlaygroundStory` util

Reply:
What should it include?

Reply:
The condition that returns boolean if it's the playground story, but it's really minor

Reply:
on both `manager` and the hook, I need to do calculations (resolve, etc) to know what's the consumer's playground story id.
So it would be a util that is exposed to storybook's api (manager-api)